provider/gce/environ_network.go

```diff
@@ -68,20 +73,20 @@ func (e *environ) getMatchingSubnets(subnetIds IncludeSet, zones []string) ([]ne
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	networks, err := e.networkIdsByURL()
+	networks, err := e.networksByURL()
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 	var results []network.SubnetInfo
 	for _, subnet := range allSubnets {
-		networkId, ok := networks[subnet.Network]
+		netwk, ok := networks[subnet.Network]
```

Comment:
`network` clashes with `github.com/juju/juju/network` which I use just below - I could rename the import instead?